onnx_tf/handlers/backend/eye_like.py

```diff
@@ -0,0 +1,56 @@
+import copy
+import numpy as np
+import tensorflow as tf
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+from onnx_tf.handlers.handler import tf_func
+
+
+@onnx_op("EyeLike")
+@tf_func(tf.eye)
+class EyeLike(BackendHandler):
+
+  @classmethod
+  def version_9(cls, node, **kwargs):
+
+    attrs = copy.deepcopy(node.attrs)
+    inp = kwargs["tensor_dict"][node.inputs[0]]
+    dtype = attrs["dtype"] if "dtype" in attrs else inp.dtype
+    offset = attrs["k"] if "k" in attrs else 0
+    shape = inp.shape.as_list()
+    # remove all attrs
+    attrs.pop("dtype", None)
+    attrs.pop("k", None)
+
+    if offset == 0:
+      shape = tf.constant(shape, dtype=tf.int32)
+      attrs["num_rows"] = shape[0]
+      attrs["num_columns"] = shape[1]
+      attrs["dtype"] = dtype
+      return [
+          cls.make_tensor_from_onnx_node(
+              node, inputs=[], attrs=attrs, **kwargs)
+      ]
+
+    else:
+      max_eye_shape_1 = shape[1] if offset > 0 else shape[0]
+      max_eye_shape_2 = shape[0] if offset > 0 else shape[1]
+      if abs(offset) > max_eye_shape_1:
+        offset = max_eye_shape_1 * np.sign(offset)
+      abs_offset = abs(offset)
+      eye_shape = min(max_eye_shape_1 - abs_offset, max_eye_shape_2)
+      tensor = tf.eye(eye_shape, num_columns=eye_shape, dtype=dtype)
+      if offset > 0:
+        tb_paddings = [0, shape[0] - eye_shape]
+        lr_paddings = [offset, shape[1] - offset - eye_shape]
+      else:
+        tb_paddings = [abs_offset, shape[0] - abs_offset - eye_shape]
+        lr_paddings = [0, shape[1] - eye_shape]
+      paddings = tf.constant([tb_paddings, lr_paddings], dtype=tf.int32)
+      attrs["paddings"] = paddings
+      attrs["constant_values"] = tf.cast(0, dtype)
```

Comment:
Hi Winnie, this is an attribute, so i don't think you should use tf.cast since it returns a tensor.
Regardless, I think you can simply ignore this field since tf.pad uses 0 as the default value to pad.